app/containers/RoomHeader/RoomHeader.stories.tsx

```
@@ -101,3 +101,32 @@ export const Themes = () => (
 		<ThemeStory theme='black' />
 	</>
 );
+
+const futureExpiry = new Date(Date.now() + 3600000).toISOString();
+
+export const DM_Status = () => (
+	<>
+		<HeaderExample title={() => <RoomHeader title='John Doe' type='d' roomUserId='user1' subtitle='Online' status='online' />} />
+		<HeaderExample title={() => <RoomHeader title='John Doe' type='d' roomUserId='user2' subtitle='Away' status='away' />} />
+		<HeaderExample title={() => <RoomHeader title='John Doe' type='d' roomUserId='user3' subtitle='Busy' status='busy' />} />
+		<HeaderExample title={() => <RoomHeader title='John Doe' type='d' roomUserId='user4' subtitle='Offline' status='offline' />} />
```

ESLint

[error] Replace `·title={()·=>·<RoomHeader·title='John·Doe'·type='d'·roomUserId='user4'·subtitle='Offline'·status='offline'·/>}·` with `⏎↹↹↹title={()·=>·<RoomHeader·title='John·Doe'·type='d'·roomUserId='user4'·subtitle='Offline'·status='offline'·/>}⏎↹↹`

(prettier/prettier)